### PR TITLE
Html css popup

### PR DESCRIPTION
**Pull request by BlackTowerKing**
### Html css popup

[Demo](https://blacktowerking.github.io/html-css-popup/)
[Code base](https://github.com/BlackTowerKing/html-css-popup)

Please, review.

### CODE REVIEW

File: submissions/BlackTowerKing/html-css-popup/css/style.css [hunk 1]
```diff
@@ -0,0 +1,229 @@
...
+    background-color: lightgrey;
+}
+
+.profile {
+    height: 40px;
+}
+
+/*End Menu*/
+
+/*Start Popup*/
```
Comment (smillims) [inline submissions/BlackTowerKing/html-css-popup/css/style.css:97]: It's empty line of your code. I recommend avoid comments like "header", "main" and etc.
Comments need for explaining part of your code, for example I'm understand what's mean 'header' or 'nav' so here it's comments redundant.

**Comment by OleksiyRudenko** [comment]
Please check and follow the submission instructions.
image

File: submissions/BlackTowerKing/html-css-popup/index.html [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+                <li class="top-menu__item">
+                    <a class="menu-link" href="#">Mail</a>
+                </li>
+                <li class="top-menu__item">
+                    <a class="menu-link" href="#">Images</a>
+                </li>
+                <li class="top-menu__item popup">  
+                    <input id="show-popup__btn" class="show__submenu-popup" type="checkbox">
+                    <label class="show-popup__label" for="show-popup__btn">
+                        <img class="menu-button" src="./img/popup-button.png" alt="popup-button">
```
Comment (smillims) [inline submissions/BlackTowerKing/html-css-popup/index.html:24]: Please styling focus of your popup menu

File: submissions/BlackTowerKing/html-css-popup/css/style.css [hunk 3]
```diff
@@ -0,0 +1,229 @@
...
+}
+table {
+	border-collapse: collapse;
+	border-spacing: 0;
+}
+a {
+    text-decoration: none;
+}
+
+/* End Reset css*/
```
Comment (smillims) [inline submissions/BlackTowerKing/html-css-popup/css/style.css:51]: I recommend move this code to another file and connect it to html for minimize your codebase

**Review by smillims — COMMENTED** [review]
Hey, good work! But I find some problems here, let's correct it together

File: submissions/BlackTowerKing/html-css-popup/css/style.css (hunk 3 above)
Comment (BlackTowerKing) [reply submissions/BlackTowerKing/html-css-popup/css/style.css]: Is it fine if I left in file css only this part 
html, body, div, span, applet, object, iframe,
h1, h2, h3, h4, h5, h6, p, blockquote, pre,
a, abbr, acronym, address, big, cite, code,
del, dfn, em, img, ins, kbd, q, s, samp,
small, strike, strong, sub, sup, tt, var,
b, u, i, center,
dl, dt, dd, ol, ul, li,
fieldset, form, label, legend,
table, caption, tbody, tfoot, thead, tr, th, td,
article, aside, canvas, details, embed, 
figure, figcaption, footer, header, hgroup, 
menu, nav, output, ruby, section, summary,
time, mark, audio, video {
	margin: 0;
	padding: 0;
	border: 0;
	font-size: 100%;
	font: inherit;
	vertical-align: baseline;
}

a {
    text-decoration: none;

Comment (smillims) [reply submissions/BlackTowerKing/html-css-popup/css/style.css]: Your are free in your choice, In this example I'm just recommend

Comment (BlackTowerKing) [reply submissions/BlackTowerKing/html-css-popup/css/style.css]: thanks I just remove unuseful styles

File: submissions/BlackTowerKing/html-css-popup/style.css [hunk 4]
```diff
@@ -0,0 +1,213 @@
...
+}
+
+.popup-btn {
+    width: 100%;
+    display: inline-flex;
+    align-items: center;
+}
+
+.btn {
+    width: 100%;
```
Comment (lenadgit) [inline submissions/BlackTowerKing/html-css-popup/style.css:152]: could you please add a property for btn  `cursor: pointer;`

**Review by lenadgit — CHANGES_REQUESTED** [review]

File: submissions/BlackTowerKing/html-css-popup/style.css [hunk 5]
```diff
@@ -0,0 +1,213 @@
...
+    background-color: rgba( 0, 0, 0, 0.2);
+    box-shadow: 5px 5px 10px  rgb(129, 128, 128);
+    font-family: Arial, Helvetica, sans-serif;
+}
+
+.popup-item {
+    text-align: center;
+    width: 33%;
+    margin-bottom: 10px;
+
```
Comment (lenadgit) [inline submissions/BlackTowerKing/html-css-popup/style.css:116]: could you please remove an empty line

**Review by lenadgit — CHANGES_REQUESTED** [review]

**Comment by lenadgit** [comment]
Cool effects for icons! really creative work

**Review by lenadgit — APPROVED** [review]

**Pull request merged**